### ISSUE TITLE
React-native run-android

### ISSUE THREAD

**Issue by alwinv135** [Group-Number-7/gpsrpg#9]
It takes 2 times to run react-native run-android in order for the app to start running

**Comment by andrewadams32** [comment]
Yes, some sort of bug with the firebase sdk. Not sure if there is something on our part we can do to fix this

**Comment by alwinv135** [comment]
Ohh okay

**Comment by GhayoorUlHaq** [comment]
I just checked running this app but first `npm start` in one terminal and then `react-native run-android` in other terminal.
App installed and loaded successfully without running `react-native run-android` second time.
But there error after loading _No Firebase App '[DEFAULT]' has been created - call firebase.initializeApp()_
